src/gsy_e/models/strategy/__init__.py

```diff
@@ -380,7 +380,7 @@ def can_offer_be_posted(
 
         posted_energy = (offer_energy
```

Comment:
Very nit, but in order to be 100% clear and to not fall into similar bugs in the future, I would rename this variable to posted_and_traded_offer_energy

Reply:
I agree and will change it.